etherman/etherman.go

```diff
@@ -1553,11 +1561,18 @@ func (etherMan *Client) EthBlockByNumber(ctx context.Context, blockNumber uint64
 
 // GetLatestBatchNumber function allows to retrieve the latest proposed batch in the smc
 func (etherMan *Client) GetLatestBatchNumber() (uint64, error) {
+	var latestBatchNum uint64
 	rollupData, err := etherMan.RollupManager.RollupIDToRollupData(&bind.CallOpts{Pending: false}, etherMan.RollupID)
 	if err != nil {
-		return 0, err
+		log.Warn("error getting latestBatchNum from rollupManager. Trying old zkevm smc... Error: ", err)
```

Comment:
Using old contracts are going to fill logs with Warning, maybe we can decide which one to use (configuration or in runtime) and just call to the right one